routes/items.js

```diff
@@ -35,4 +35,15 @@ router.post( '/:id/completed', ( request, response ) => {
     .catch( error => response.json({ success: false, id, message: error.message }))
 })
 
+router.post( '/:id/edit', ( request, response ) => {
+  const Item = request.app.get( 'models' ).Item
+
+  const { id } = request.params
+  const { title, description } = request.body
```

Comment:
We should probably do some validation of the `request.body` to ensure we're getting only valid fields (and sending only valid fields on to the database).